libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInvoice.kt

```diff
@@ -47,8 +50,13 @@ class LnurlPayInvoiceJob(
         var request: LnurlInvoiceRequest? = null
         try {
             request = Json.decodeFromString(LnurlInvoiceRequest.serializer(), payload)
-            val nodeState = breezSDK.nodeInfo()
-            if (request.amount < 1000UL || request.amount > nodeState.inboundLiquidityMsats) {
+            val ofpResp =
+                breezSDK.openChannelFee(OpenChannelFeeRequest(amountMsat = request.amount))
+            val feeLimitMsats: ULong = config.autoChannelSetupFeeLimitMsats
+            val isFeeWithinLimits = ofpResp.feeMsat?.let { it != 0UL && it > feeLimitMsats }
```

Comment:
Is within limits if the ofp fee equals 0 or ofp fee is less or equal to the fee limit
```suggestion
            val isFeeWithinLimits = ofpResp.feeMsat?.let { it == 0UL || it <= feeLimitMsats }
```

Reply:
Addressed both suggestions: [Correct isFeeWithinLimits condition](https://github.com/breez/breez-sdk/pull/833/commits/19bab0c0c38ad863bb81db040cb22ed69910fa1b)